src/components/shared/Map/MapPointer.tsx

```diff
@@ -10,17 +10,48 @@ interface MapPointerProps {
   center: [number, number];
   radius: number;
   text: string;
-  color: string;
   sensorData: SensorData;
 }
 
-function MapPointer({
-  center,
-  radius,
-  text,
-  color,
-  sensorData
-}: MapPointerProps) {
+const colors = ['#6ebc2a', '#c4d83f', '#ffe500', '#f1aa41', '#b91414'];
+
+type PollutionLevels = Array<{
+  name: string;
+  values: number[];
+}>;
+
+const pollutionLevels: PollutionLevels = [
+  {
+    name: 'PM10',
+    values: [0, 40, 80, 120, 300]
+  },
+  {
+    name: 'PM2.5',
+    values: [0, 25, 50, 100, 300]
+  }
+];
+
+function colorCircle(sensorData: SensorData, pollutionName: string): string {
```

Comment:
```suggestion
function getCircleColor(sensorData: SensorData, pollutionName: PollutionType): string {
```

You can create a `PollutionType` type that is a union of 2 supported pollution types:

`type PollutionType = 'PM2.5' | 'PM10'`

It will be less confusing than allowing any string.